git/exc.py

```diff
@@ -5,7 +5,34 @@
 # the BSD License: https://opensource.org/license/bsd-3-clause/
 """ Module containing all exceptions thrown throughout the git package """
 
-from gitdb.exc import (  # noqa: @UnusedImport
+__all__ = [
+    # Defined in gitdb.exc:
+    "AmbiguousObjectName",
+    "BadName",
+    "BadObject",
+    "BadObjectType",
+    "InvalidDBRoot",
+    "ODBError",
+    "ParseError",
+    "UnsupportedOperation",
+    # Introduced in this module:
+    "GitError",
+    "InvalidGitRepositoryError",
+    "WorkTreeRepositoryUnsupported",
+    "NoSuchPathError",
+    "UnsafeProtocolError",
+    "UnsafeOptionError",
+    "CommandError",
+    "GitCommandNotFound",
+    "GitCommandError",
+    "CheckoutError",
+    "CacheError",
+    "UnmergedEntriesError",
+    "HookExecutionError",
+    "RepositoryDirtyError",
```

Comment:
Maybe these should be alphabetized? But it was easier for me to verify that they were all there when listed in the same order they are defined in this same module, and I suspect others reading and working with the code may find this easier to verify as well, so I've kept them in this order.

Reply:
Both orders have their value, and I absolutely leave it to you to decide which one to prefer. Let's keep it the way it is for now then.